ECOSYSTEM.md

```diff
@@ -14,12 +14,19 @@ Partners are companies that are significantly invested in React Native and demon
 
 Partners think of React Native as a product; they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications.
 
-Baseline responsibilities of partners include:
-* Attending monthly meeting
-* Contributing to the release process. Examples include being a [community releaser](https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2-community-releaser), testing new releases, technical support for release issues.
-* Engage on the core contributor Discord
+### Application process
+To become a React Native partner, an existing partner needs to refer and champion your application. Partners will undergo a 3 month incubating period, after which Partners will vote to convert to full membership.
 
-Our current partners and other areas of ownership:
+Partnership is not a status symbol, it is a commitment to invest significant resources into improving React Native.
+
+Maintaining partner status requires consistently meeting the baseline responsibilities, including:
+* Attending monthly meeting.
+* Contributing to the release process.
+* Engaging in the core contributor Discord.
+
+Examples of contributing to the release include being a [community releaser](https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2-community-releaser), testing new releases, and technical support for release issues.
```

Comment:
Not totally related to this doc but I think is a good idea to clarify in the release role doc that becoming part of the release crew is a multi month commitment until the baton is passed to the next crew. Additionally, the release crew is responsible for all active releases in the support window, not just the upcoming one.